internal/pkg/otel/samples/darwin/hostmetrics_resources.yml

```
@@ -0,0 +1,34 @@
+receivers:
+  # Receiver for CPU, Disk, Memory, and Filesystem metrics
+  hostmetrics/system:
+    collection_interval: 30s
+    scrapers:
+      disk:
+      filesystem:
+      cpu:
+      memory:
+
+processors:
+  resourcedetection/system:
+    detectors: ["system"]
+    system:
+      hostname_sources: ["os"]
+
+exporters:
+  # Exporter to print the first 5 logs/metrics and then every 1000th
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 1000
+
+  # Exporter to send logs and metrics to Elasticsearch
+  elasticsearch/bulk:
+    endpoints: [<<ES_ENDPOINT>>]
+    api_key: <<ES_API_KEY>>
```

Not sure if this is a good idea, but maybe use environment variables here? :thinking: 

```suggestion
    endpoints: [${env:ELASTIC_ENDPOINT}]
    api_key: ${env:ELASTIC_API_KEY}
```

---

@flash1293 can you alter your script to replace variable like this so we have at least something usefull in our configs?